app/controllers/podcast_metrics_controller.rb

```diff
@@ -1,112 +1,273 @@
 class PodcastMetricsController < ApplicationController
   before_action :set_podcast
+  before_action :set_date_range, only: %i[show downloads episodes uniques]
 
   def show
-    authorize @podcast, :show?
+  end
+
+  def downloads
```

Comment:
in the previous PR, all queries were made in the `show` route, and now all the main downloads tabs have been moved to their own respective route/action. this is the main `downloads` route, which is for the podcast's total downloads within the selected date range.